rgrow/src/models/sdc1d.rs

```diff
@@ -1111,6 +1247,7 @@ impl TileBondInfo for SDC {
 // Here is potentially another way to process this, though not done.  Feel free to delete or modify.
 
 use std::hash::Hash;
+use crate::system::Orientation::WE;
```

Comment:
We probably shouldn't be having use statements way down here unless there's some specific reason to?